react/features/overlay/middleware.ts

```diff
@@ -102,10 +98,11 @@ StateListenerRegistry.register(
                 ...error,
                 ...getErrorExtraInfo(getState, error)
             });
-        }
-
-        if (NON_OVERLAY_ERRORS.indexOf(error.name) === -1 && typeof error.recoverable === 'undefined') {
```

Comment:
Why did you remove the check? Wouldn't this now show the reload overlay also for non-fatal errors?

Reply:
I thought for those errors we don't show the reload overlay. Gonna revert this.